src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -282,7 +292,7 @@ public String getInvalidityInfoForEmail(String email) {
         String sanitizedValue = SanitizationHelper.sanitizeForHtml(email);
 
         if (email.isEmpty()) {
-            return getPopulatedErrorMessage(EMAIL_ERROR_MESSAGE, email, EMAIL_FIELD_NAME, REASON_EMPTY,
+            return getPopulatedErrorMessage(EMAIL_ERROR_MESSAGE_EMPTY_STRING, email, EMAIL_FIELD_NAME, REASON_EMPTY,
```

Comment:
@leeyimin I don't understand why there is a need to create a new method just for `EMPTY_STRING`. I think we should revert [this](https://github.com/TEAMMATES/teammates/pull/7783#discussion_r127907532) change and use `getPopulatedErrorMessage` only.